plugins/GSdx/GSRendererHW.cpp

```diff
@@ -502,7 +502,7 @@ void GSRendererHW::Draw()
 	context->ZBUF.ZMSK = zm != 0;
 
 	// A couple of hack to avoid upscaling issue. So far it seems to impacts mostly sprite
-	if ((m_upscale_multiplier > 1) && (m_vt.m_primclass == GS_SPRITE_CLASS)) {
+	if ((m_upscale_multiplier > 1) && (m_upscale_multiplier < 7) && (m_vt.m_primclass == GS_SPRITE_CLASS)) {
```

Comment:
formerly, you used if ((m_upscale_multiplier > 1) which means value should be greater than custom, 1 was for custom. so I used <7 there, if the hacks are intended also for custom resolution then I'll remove that condition.